core/src/modules/statusReporters.ts

```diff
@@ -437,7 +437,7 @@ export namespace FinalSummaryReporters {
         await run.updateTotals();
         const schedule = await Schedule.findByPk(run.creatorId);
 
-        if (schedule) source = await schedule.$get("source");
+        if (schedule) source = await schedule.$get("source", { scope: null });
```

Comment:
👍  Makes sense - Some exports may have been sent in the case where something was deleted